src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -909,10 +909,10 @@ internal override void Complete()
                         ActionPreference.SilentlyContinue,
                         out _);
 
-                    const string errorId = nameof(CommandBaseStrings.ProgramFailedToComplete);
+                    const string errorId = nameof(CommandBaseStrings.ProgramEndsWithNonZeroCode);
```

Comment:
Does the RFC need to be updated to reflect this change?  I like this better than `ProgramFailedToComplete` but would it be better as either `ProgramEndedWithNonZeroCode` or `ProgramExitedWithNonZeroCode`?  Somehow the "present tense" of `EndsWIth` strikes me as odd.

Reply:
`ProgramExitedWithNonZeroCode` looks good to me :)